src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: $main-font-family;
```

Comment:
The variable `$main-font-family` is used here. Ensure this variable is defined in your SCSS files to avoid errors during compilation.